packages/react-native-gesture-handler/src/v3/detectors/HostGestureDetector.tsx

```diff
@@ -1,3 +1,4 @@
 import RNGestureHandlerDetectorNativeComponent from '../../specs/RNGestureHandlerDetectorNativeComponent';
+export { NativeProps as RNGestureHandlerDetectorNativeComponentProps } from '../../specs/RNGestureHandlerDetectorNativeComponent';
```

Comment:
```suggestion
export type { NativeProps as RNGestureHandlerDetectorNativeComponentProps } from '../../specs/RNGestureHandlerDetectorNativeComponent';
```

It's only a type, right?